networkit/cpp/algebraic/test/MatricesGTest.cpp

```diff
@@ -1200,6 +1212,11 @@ TEST_F(MatricesGTest, testDimension) {
     testDimension<DenseMatrix>();
 }
 
+TEST_F(MatricesGTest, testMatrixToGraph) {
+    testMatrixToGraph<DynamicMatrix>();
+    testMatrixToGraph<CSRMatrix>();
+}
```

Comment:
There is a more elegant way to instantiate a type-parametrized test suites: http://google.github.io/googletest/reference/testing.html#TYPED_TEST_SUITE_P

Anyway, this is not something introduced by this PR. This is ok for now, but we should change it later.